app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -105,8 +118,27 @@ export class SandboxedMarkdown extends React.PureComponent<
     this.frameContainingDivRef = frameContainingDivRef
   }
 
+  private initializeMarkdownEmitter = async () => {
+    if (this.markdownEmitter !== undefined) {
+      this.markdownEmitter.dispose()
+    }
+    const { emoji, repository, markdownContext } = this.props
+    this.markdownEmitter =
+      typeof this.props.markdown !== 'string'
+        ? this.props.markdown
+        : parseMarkdown(this.props.markdown, {
+            emoji,
+            repository,
+            markdownContext,
+          })
```

Comment:
Note.. this was supposed to be part of my last PR on this (but didn't cherry pick that commit).. and since it wasn't the markdown parsing is regressed to not having custom markdown filters on development. If we were to decide not to merge this PR, I would at least need to open a PR to provide this options **before next beta.**

Reply:
Opened [this PR](https://github.com/desktop/desktop/pull/14626) in case this is not merged before next beta.

Reply:
(Other PR merged so ignore this)